app/services/push_notification_service.rb

```diff
@@ -0,0 +1,32 @@
+class PushNotificationService
+  @@server_key = 'xxx'
+  @@max_count = 2
```

Comment:
Base on fcm gem document:
<img width="914" alt="spacialdb:fcm: Ruby bindings to Firebase Cloud Messaging (FCM) for Android, iOS or Web 2019-10-21 15-47-15" src="https://user-images.githubusercontent.com/5484758/67190477-3a8a9b00-f41a-11e9-95d6-0c60071fb899.png">